tests/POW/test_POW.cpp

```diff
@@ -395,29 +395,6 @@ BOOST_AUTO_TEST_CASE(mining_and_verification_full) {
   BOOST_REQUIRE(!verifyWinningNonce);
 }
 
-BOOST_AUTO_TEST_CASE(mining_low_diffculty_time_out) {
```

Comment:
Which test is this a duplicate of?

Reply:
It is to test the time out mechanism, the mining_high_diffculty_time_out test case is enough